static/slides/javascript.html

```diff
@@ -322,25 +323,46 @@
                   ```
 
 
-                  <!-- .slide: data-background="#D8C99B" -->
+                <!-- .slide: data-background="#D8C99B" -->
+                 ## Function Hoisting 
+                  On peut hisser que les déclarations de fonction
```

Comment:
il manque un truc dans ta phrase.

Tu as le droit de mettre hoisting dans la phrase (car pas trop de traduction française).

```suggestion
                 Les déclarations de fonctions sont toujours exécuté avant le reste.
```